docs/notebooks/atmodel_tutorial.ipynb

```diff
@@ -0,0 +1,1061 @@
"M.Cekic\",\"M.Dale\",\"M.Deweese\",\"M.Dikaiakos\",\"M.Finke\",\"M.Garzon\",\"M.Gilloux\",\"M.H.Goldstein\",\"M.Hasselmo\",\"M.Herbster\",\"M.I.Jordan\",\"M.Isard\",\"M.J.Denham\",\"M.J.Rose\",\"M.Jabri\",\"M.K.Warmuth\",\"M.Kadirkamanathan\",\"M.Kearns\",\"M.Konishi\",\"M.Leroux\",\"M.M.Hayhoe\",\"M.M.Hochberg\",\"M.Mahowald\",\"M.Marchand\",\"M.Meila\",\"M.Niranjan\",\"M.Pavel\",\"M.R.Walker\",\"M.Revow\",\"M.S.Bartlett\",\"M.Schenkel\",\"M.Svensen\",\"M.V.Solodov\",\"M.Veloso\",\"M.W.Craven\",\"M.W.Pealersen\",\"M.Wattenberg\",\"MagnusRattray\",\"MagnusStensmo\",\"MahesanNiranjan\",\"MaheshVaranasi\",\"MakotoHirayama\",\"MalcolmSlaney\",\"MalikMagdon-Ismail\",\"ManceHarmon\",\"ManeeshSahani\",\"ManfredK.Warmuth\",\"ManfredOpper\",\"ManoelF.Tenorio\",\"ManoelFernandoTenorio\",\"ManuelF.Fernandez\",\"ManuelSamuelides\",\"Marc-OlivierCoppens\",\"MarcH.Cohen\",\"MarcLoinaz\",\"MarcellaA.McClure\",\"MarcelloPelillo\",\"MarcusFrean\",\"MarcusHeld\",\"MarcusHennecke\",\"MargaretSereno\",\"MarianS.Bartlett\",\"MarianStewartBartlett\",\"MarilynWalker\",\"MarinaMeila\",\"MarioBlaum\",\"MarioMarchand\",\"MarioP.Vecchi\",\"MarissaWesterfield\",\"MariusUsher\",\"Mark.RSydorenko\",\"MarkA.Gluck\",\"MarkA.Rubin\",\"MarkDerthick\",\"MarkE.Nelson\",\"MarkFanty\",\"MarkGerules\",\"MarkGluck\",\"MarkHoller\",\"MarkKvale\",\"MarkMathieson\",\"MarkOllila\",\"MarkPlutowski\",\"MarkRing\",\"MarkSaffman\",\"MarkSitton\",\"MarkW.Goudreau\",\"MarkWhite\",\"MarkZlochin\",\"MarkusLappe\",\"MarkusMeister\",\"MarkusSchenkel\",\"MarkusSvensen\",\"MarleneBehrmann\",\"MarshallFlax\",\"MarthaFarah\",\"MartinHammer\",\"MartinI.Sereno\",\"MartinJ.Johnson\",\"MartinJ.McKeown\",\"MartinJ.Wainwright\",\"MartinLades\",\"MartinMaechler\",\"MartinRiedmiller\",\"MartinRoscheisen\",\"MartinS.Glassman\",\"MartinSereno\",\"MartinStemmler\",\"MartinStetter\",\"MartineNaillon\",\"Martinede-Gerlache\",\"MarvinLuttges\",\"MarwanA.Jabri\",\"MarwanJabd\",\"MarwanJabri\",\"MaryB.Oftaway\",\"MaryE.T.Boyle\",\"MaryTabasko\",\"Masa-akiSato\",\"MasahikoHamno\",\"MasashiSugiyama\",\"MasayukiMiyamoto\",\"MasazumiKatayama\",\"MassimoSivilotti\",\"MassoudOmidvar\",\"MathiasQuoy\",\"MatsOsterberg\",\"MattMelton\",\"MatthewA.Wilson\",\"MatthewBrand\",\"MatthewJ.Beal\",\"MatthewM.Williamson\",\"MatthewN.Dailey\",\"MatthewSaffell\",\"MatthiasBurger\",\"MatthiasScholz\",\"MatthiasSeeger\",\"MattiHamalainen\",\"MauriceLee\",\"MauriceMilgram\",\"MauroBirattari\",\"MaxGarzon\",\"MaximilianRiesenhuber\",\"MazinRahim\",\"MelanieMitchell\",\"MenasheDornay\",\"Meng-JangLin\",\"MicahS.Siegel\",\"MichaelA.Glover\",\"MichaelBrownlow\",\"MichaelBurl\",\"MichaelC.Crair\",\"MichaelC.Mozer\",\"MichaelChuang\",\"MichaelCohen\",\"MichaelD.Revow\",\"MichaelDuff\",\"MichaelE.Hasselmo\",\"MichaelE.Levemon\",\"MichaelE.Tipping\",\"MichaelFinke\",\"MichaelFleisher\",\"MichaelG.Dyer\",\"MichaelG.Paulin\",\"MichaelGasser\",\"MichaelHasselmo\",\"MichaelHormel\",\"MichaelHumphreys\",\"MichaelI.Jordan\",\"MichaelI.Miller\",\"Michael
```

Comment:
Make the statement about radius in bold above the plot. it took me a while to find this hidden comment